packages/sources/crypto-api-v2/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@chainlink/crypto-api-v2-adapter",
```

The data provider is called "CryptoAPIs", so the package name (and path) should be "cryptoapis-v2".

```suggestion
  "name": "@chainlink/cryptoapis-v2-adapter",
```

---

Can I just change the package name and this file here or are there other places I need to change it too?

---

There are a few other files (like tsconfig) that reference this (or at least the path). You might have to do a search for all references of that string and replace it.